test/jdk/java/net/httpclient/HttpRequestBuilderTest.java

```diff
@@ -295,13 +297,7 @@ static void method(String name,
             throw new AssertionError("failed: " + name
                     + ". Unexpected body processor for GET: "
                     + request.bodyPublisher().get());
-
-        if (expectedMethod.equals(method)) {
-            System.out.println("success: " + name);
-        } else {
-            throw new AssertionError("failed: " + name
-                    + ". Expected " + expectedMethod + ", got " + method);
-        }
+        assertEquals(method, expectedMethod, "Method");
```

Comment:
Same here - This should be:

```
assertEquals(expectedMethod, method, "Method");
```

Reply:
Fixed in 9f7b054b0e96244aea374dbf2555f2a03f61cab0.